include/vcpkg/base/util.h

```diff
@@ -130,12 +130,57 @@ namespace vcpkg::Util
         return ret;
     }
 
+    template<class Container, class T>
+    void erase_remove(Container& cont, const T& el)
+    {
+        cont.erase(std::remove(cont.begin(), cont.end(), el), cont.end());
+    }
     template<class Container, class Pred>
     void erase_remove_if(Container& cont, Pred pred)
     {
         cont.erase(std::remove_if(cont.begin(), cont.end(), pred), cont.end());
     }
 
+    // 0th is the first occurence
+    // so find_nth({1, 2, 1, 3, 1, 4}, 1, 2)
+    // returns the 1 before the 4
+    template <class R, class V>
+    auto find_nth(R&& r, const V& v, size_t n)
+    {
+        using std::begin;
+        using std::end;
+
+        auto last = end(r);
+        auto current = std::find(begin(r), last, v);
```

Comment:
I doubt you care but the STL name for this would be "first" rather than "current". ([first,last) is for ranges, [begin(), end()) for containers over there).